.circleci/make-rpm-packages.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -eux
+
+echo "PARAM_RELEASE: $PARAM_RELEASE"
+echo "PARAM_DIST: $PARAM_DIST"
+
+if test "x$PARAM_RELEASE" = "x"; then
+    echo "Env variable PARAM_RELEASE is not set! For example PARAM_RELEASE=8, for CentOS 8"
+    exit 1
+elif test "x$PARAM_DIST" = "x"; then
+    echo "Env variable PARAM_DIST is not set! For example PARAM_DIST=centos"
+    exit 1
+fi
+
+if [ "$PARAM_DIST" = centos ]; then
+  if [ "$PARAM_RELEASE" = 8 ]; then
+      dnf install -y 'dnf-command(config-manager)'
+      yum config-manager --set-enabled PowerTools
+      yum install -y diffutils python3-sphinx
+  else
+      yum install -y python-sphinx
+  fi
+fi
+
+yum install -y epel-release
+yum install -y automake jemalloc-devel git libedit-devel libtool libunwind-devel make pcre-devel python3 sudo rpm-build yum-utils
```

Comment:
this is wrong, the `yum-builddep` is responsible for installing the dependencies listed in the spec file.

(you do need `epel-release` and `PowerTools` repos to be enabled though)

Reply:
I'll move as many as possible to `redhat/varnish.spec` but `automake` needs to be installed here because of
   `VERSION=$("$DIST_DIR"/configure --version | awk 'NR == 1 {print $NF}')$WEEKLY` 
at line 54

Reply:
I don't think you do, `configure` should just be a `shell` script

Reply:
this hasn't been fixed yet. And most deps should be handled in the spec, not here.